tasks/check-data-keys/tests/test-check-data-keys-fail-unsupported-system.yaml

```diff
@@ -87,6 +88,7 @@ spec:
           value:
             - releaseNotes
             - unsupported
+            - random
```

Comment:
why? Isn't unsupported sufficient?

Reply:
I wanted to check if a user had multiple unsupported systems that would output all of them, not just one.